internal/cmd/local/command_kafka_start.go

```diff
@@ -58,31 +65,35 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 	}
 	defer out.Close()
 
-	buf := new(strings.Builder)
-	_, err = io.Copy(buf, out)
-	if err != nil {
-		return err
-	}
+	scanner := bufio.NewScanner(out)
+	for scanner.Scan() {
+		var response ImagePullResponse
+		text := scanner.Text()
 
-	for _, ss := range strings.Split(buf.String(), "\n") {
-		var output imagePullOut
-		if err := json.Unmarshal([]byte(ss), &output); err != nil {
-			continue
+		err := json.Unmarshal([]byte(text), &response)
+		if err != nil {
+			return err
 		}
+
 		var inBlacklist bool
 		for _, s := range statusBlacklist {
-			if output.Status == s {
+			if response.Status == s {
 				inBlacklist = true
```

Comment:
Nitpick:
This block can be condensed with `inBlacklist := types.Contains(statusBlacklist, response.Status)`
You could also roll it into the if below: `if !types.Contains(statusBlacklist, response.Status) {`

Reply:
could you be more specific? i can't find `Contains` in `go/types` package

Reply:
Sorry, I meant our own types package `"github.com/confluentinc/cli/internal/pkg/types"`. It's in `pkg/types/slice.go`

Reply:
i'm now printing out all outputs so no blacklist is needed now! thanks for the advice though

Reply:
Nice!